lib/MetaCPAN/Web/Controller/Recent.pm

```diff
@@ -17,7 +17,11 @@ sub index : Path : Args(0) {
         ->recent( $req->page, $page_size, $req->params->{f} || 'l' )->recv;
     my $latest = [ map { $_->{fields} } @{ $data->{hits}->{hits} } ];
     single_valued_arrayref_to_scalar($latest);
-    $c->res->last_modified( $latest->[0]->{date} ) if (@$latest);
+
+    $c->add_surrogate_key('RECENT');
+    $c->browser_max_age('1m');
+    $c->cdn_max_age('1m');    # tough to paging for now
```

Comment:
You _might_ get something appearing at the end of page 1, and the start of page 2, if you happen to hit them at different - BRAIN WAVE! - api now purges 'DIST_UPDATES' any time there is an update to any dist, so we can cache this for a year - pull request will reflect this shortly